js/taskjobs.js

```diff
@@ -712,7 +712,7 @@ taskjobs.update_logs = function (data) {
          var targets_cpt = 0;
 
          if(job_v.targets.length === 0) {
-            $(targets_selector).html('The preparation of the task did not return any targets');
+            $(targets_selector).html(__('The preparation of the task did not return any targets', 'glpiinventory'));
```

Comment:
From `js` file call to `__('my string', 'context')` is not possible as this is a PHP function

Reply:
Hello @stonebuzz, it’s now possible since GLPI 9.5.0. I tested it and it’s working as expected. All PHP functions was backported to JavaScript: https://glpi-developer-documentation.readthedocs.io/en/master/devapi/translations.html

Reply:
you're right (it's been so long since I've done gettext in js :smile:)

you can ignore this comment